tools/scripts/utils.mjs

```diff
@@ -566,6 +611,39 @@ export const log = Object.assign(
   }
 );
 
+export const testThis = async (test) => {
+  // calculate the time it takes to run the test
+  const start = Date.now();
+
+  const { status, message } = await test.fn();
+
+  let errorIsThrown = false;
+
+  try {
+    const end = Date.now();
+
+    const time = end - start;
+
+    if (status === 200) {
+      log.green(`\t${message} (${time}ms)`);
+      return true;
```

Comment:
200 might not always mean the request was sucessful. maybe we should also check for any errors in the object?

Reply:
sorry should've properly documented this - it will also return `200` cus in the test we are relying on the custom `success` or `fail` function, so it will either return 200, or 500. If not, the test is not correctly implemented:

```
export const success = (message) => {
  return {
    status: 200,
    message,
  };
};

export const fail = (message) => {
  return {
    status: 500,
    message,
  };
};
```

Reply:
https://github.com/LIT-Protocol/js-sdk/blob/952f7abecdafc451599c6734e6a6a3c2bb0b975b/tools/scripts/utils.mjs#L20-L32